test/spec/runners/ssz_static.ex

```diff
@@ -84,14 +84,15 @@ defmodule SszStaticTestRunner do
     assert_ssz(schema, decompressed, expected, expected_root)
   end
 
-  defp assert_ssz(schema, real_serialized, real_deserialized, expected_root) do
+  defp assert_ssz(schema, real_serialized, real_deserialized, _expected_root) do
     {:ok, deserialized} = SszEx.decode(real_serialized, schema)
```

Comment:
Can we encode both with Ssz and Ssz and assert that both results are equal? For encoding and also for decoding?

We should make sure that we can switch one implementation for the other.

Reply:
it looks like there are discrepancies between bit_list and bit_vectors. The new library already decodes those elements.
For example:
```%Types.Attestation{aggregation_bits: <<0::size(2)>> ...```
and the rust implementation
```%Types.Attestation{aggregation_bits: <<4>> ....```

Reply:
oh I think the problem is the `sanitize_ssz` function :thinking:

Reply:
I am going to create a function that sanitizes the bit_list and bit_vector fields from the rust decode part

Reply:
Ok, I think that is done. Comparing the `decode` results was tricky because of bit_list and bit_vectors and the Aja.Vector fields. 
The `encoding` part worked right away. Let me know if the process done for the decode part is ok.